haskell-indexer-translate/src/Language/Haskell/Indexer/Translate.hs

```diff
@@ -30,6 +30,7 @@ module Language.Haskell.Indexer.Translate
     --
     , XRef(..)
     , AnalysedFile(..)
+    , ModuleTick(..)
```

Comment:
It's analogous to `Tick` which is a unique name for something. I agree it's not best. `Module` happens in GHC, so I would avoid the confusion if possible.